api_fhir_r4/multiserializer/mixins.py

```diff
@@ -215,11 +282,20 @@ def list(self, request, *args, **kwargs):
             model = next_serializer_data.model
             filtered_querysets[model, serializer] = next_serializer_data
 
-        page = self.paginate_queryset(list(chain(*filtered_querysets.values())))
-        data = self.__dispatch_page_data(page)
-        serialized_data = self._serialize_dispatched_data(data, dict(filtered_querysets.keys()))
-        data = self.get_paginated_response(serialized_data)
-        return data
+        if not filtered_querysets:
+            return {}
+
+        try:
+            querysets = self._join_querysets([*filtered_querysets.values()])
+            page = self.paginate_queryset(querysets)
+            data = self.__dispatch_page_data(page)
+            serialized_data = self._serialize_dispatched_data(data, dict(filtered_querysets.keys()))
+            data = self.get_paginated_response(serialized_data)
+            return data
+        except Exception as e:
+            import traceback
+            traceback.print_exc()
```

Comment:
Should be logger instead of traceback? Moreover I see you assumed that in that section will be exception raised. And what about another block above of this try catch?